.eslintrc.json

```diff
@@ -1,3 +1,15 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": ["next", "next/core-web-vitals", "eslint:recommended"],
+  "globals": {
+    "React": "readonly"
+  },
+  "rules": {
+    "no-unused-vars": [
```

Comment:
Warns when there's a specific variable in the codebase which is not being used. If we, at some point, need to add a variable that is not going to be used at the moment, but we'll use it in the future, we can tell eslint to ignore that specific unused variable by prefixing its name with `_`. Example:

```
const _someUnusedVariable = 5;
```